code/modules/surgery/organs/autoimplanter.dm

```diff
@@ -0,0 +1,33 @@
+/obj/item/device/autoimplanter
+	name = "autoimplanter"
+	desc = "A device that automatically injects a cyber-implant into the user without the hassle of extensive surgery. It has a slot to insert implants and a screwdriver slot for removing accidentally added implants."
+	icon_state = "gangtool-white"
+	item_state = "walkietalkie"
+	w_class = 2
+	var/obj/item/organ/internal/cyberimp/storedorgan
+
+/obj/item/device/autoimplanter/attack_self(mob/user, obj/item/I)//when the object it used...
```

Comment:
shit you're right, i believe we figured this out earlier but i forgot to tweak it in the finalized code